### PR TITLE
Rest stat vars union documentation

### PR DESCRIPTION
**Pull request by KilimAnnejaro**
Resolves https://github.com/datacommonsorg/docsite/issues/149.

### CODE REVIEW

**Review by beets — APPROVED** [review]
thanks for adding this!

**Review by beets — APPROVED** [review]
thanks for adding this!

File: api/rest/union.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+{% endtabs %}
+
+</div>
+
+#### Response
+
+```json
+{
+  "statVars": {
```
[the remark this answers is not in the document]
Comment (shifucun) [reply api/rest/union.md:63]: Sent out a fix for this https://github.com/datacommonsorg/mixer/pull/531

**Comment by beets** [comment]
Thanks for the quick fix Bo! I'll update the doc once that's been pushed.

**Pull request closed without merging**